### ISSUE TITLE
Link to Normandy Recipe ID

### ISSUE THREAD

**Issue by shell1** [mozilla/experimenter#911]
Connecting to the JSON end point with recipe ID number to go fetch that recipe info for viewing from inside Experimenter.

If folks have the VPN - believe this will work.  So meets test cases for improving administration.

If folks do not have the VPN - I suspect this is where we hit the issue mython reported - on showing the non-VPN level view: https://github.com/mozilla/delivery-console/issues/298

**Comment by mythmon** [comment]
The JSON endpoint will work regardless of VPN status. The URL to use would be `https://normandy.cdn.mozilla.net/api/v3/recipe/{id}/`.

mozilla/delivery-console#298 is only relevant for linking directly to the Delivery Console UI, which seems separate from this.

**Comment by shell1** [comment]
👍  thank you.  will update original comment based on what you said.  Filing a separate issue, that will be blocked on #298 on connection to the console.  That is pretty far out in the backlog after other stuff - but just so we keep it for future project work.

**Comment by jaredkerim** [comment]
Just to add some additional clarity, I believe this is asking to add a link to the Experiment Detail page to the Delivery Console page for that experiments Normandy recipe.  Once we do #910 and #498 then the experiment will have the recipe ID and adding this link to the link bar on the detail page should be straight forward.

**Comment by jaredkerim** [comment]
I actually ended up doing this inside #498 so I'm closing this.
